code/modules/clothing/spacesuits/rig_modules/rig_modules.dm

```diff
@@ -143,10 +143,16 @@
 	name = "plasma-proof sealing authority"
 	desc = "Brings the suit it is installed into up to plasma environment standards."
 	active_power_usage = 5
+	var/already_has = FALSE //what kind of shitlang uses all cap booleans?
```

Comment:
Stupid comment to add given that this is a repo convention to use a define, it's not part of the language